satellite/repair/repairer/repairer.go

```diff
@@ -49,16 +49,15 @@ type Service struct {
 }
 
 // NewService creates repairing service
-func NewService(log *zap.Logger, queue queue.RepairQueue, config *Config, interval time.Duration, concurrency int, transport transport.Client, metainfo *metainfo.Service, orders *orders.Service, cache *overlay.Service) *Service {
-	client := ecclient.NewClient(log.Named("ecclient"), transport, config.MaxBufferMem.Int())
-	repairer := NewSegmentRepairer(log.Named("repairer"), metainfo, orders, cache, client, config.Timeout, config.MaxExcessRateOptimalThreshold)
+func NewService(log *zap.Logger, queue queue.RepairQueue, config *Config, transport transport.Client, metainfo *metainfo.Service, orders *orders.Service, cache *overlay.Service, satelliteSignee signing.Signee) *Service {
+	repairer := NewSegmentRepairer(log.Named("repairer"), metainfo, orders, cache, transport, config.Timeout, config.MaxExcessRateOptimalThreshold, satelliteSignee)
```

Comment:
I would make `SegmentRepairer` an argument to `NewService`. It will reduce number of parameters and will decouple repairer from service a little bit.